### DISCUSSION TITLE
你们的官网是不是挂了？

### DISCUSSION THREAD

**Discussion by egege** [SonicCloudOrg/sonic-server#443] [General]
https://sonic-cloud.cn/
<img width="875" alt="截屏2024-05-21 17 22 34" src="https://github.com/SonicCloudOrg/sonic-server/assets/23636455/15e9f234-2faa-43fc-adb0-6a2d753b1dd6">

**Comment by liupeiyu** [comment]
![WechatIMG139](https://github.com/SonicCloudOrg/sonic-server/assets/8373743/fc76fa2c-3f95-4aa5-ab9f-1362165013f8)

**Comment by ZhouYixun** [comment]
https://soniccloudorg.github.io/
